instaclustr/resource_bundle_user.go

```diff
@@ -0,0 +1,166 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBundleUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBundleUserCreate,
+		Read:   resourceBundleUserRead,
+		Update: resourceBundleUserUpdate,
+		Delete: resourceBundleUserDelete,
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"password": {
```

Comment:
This should be marked as `Sensitive: true` as per [this change](https://github.com/instaclustr/terraform-provider-instaclustr/commit/b65d5b235c29165c9dbad7589271bb1407698889)

Reply:
done